string nodeName
---
uint16 nodeId
